src/lifecycle/jvm_manager.cpp

```diff
@@ -70,17 +71,19 @@ void JvmManager::initialize_or_get_jvm(void* lib_handle, JvmUserConfiguration& u
     LOG_VERBOSE("Starting JVM ...");
     JNIEnv* jni_env {nullptr};
 
-    jint result {get_create_jvm_function(lib_handle)(&java_vm, reinterpret_cast<void**>(&jni_env), &args)};
+    CreateJavaVM func {get_create_jvm_function(lib_handle)};
+    JVM_ERR_FAIL_COND_V_MSG(func == nullptr, false,"Failed to obtain JNI_CreateJavaVM symbol from dynamic library!");
+    jint result {func(&java_vm, reinterpret_cast<void**>(&jni_env), &args)};
 
     // Set std::local::global to value it was before creating JVM.
     // See https://github.com/utopia-rise/godot-kotlin-jvm/issues/166
     // and https://github.com/utopia-rise/godot-kotlin-jvm/issues/170
-#ifndef NO_USE_STDLIB
+#ifndef NO_USE_STDLIBss
```

Comment:
`NO_USE_STDLIBss` ? why `ss`?

Reply:
Because I'm a grammar nazi obviously.
But just a mistake I probably had the wrong window selected when trying to type something and didn't pay attention (having to often switch between Clion and Intellij).